apps/ledger-live-desktop/src/main/index.js

```diff
@@ -75,6 +75,23 @@ app.on("ready", async () => {
     await installExtensions();
   }
 
+  /**
+   * Clears the session’s HTTP cache
+   * Used to remove third party cached auth tokens, among other things
+   */
+  ipcMain.handle("clearStorageData", () => {
```

Comment:
Does this get cleared on every startup of the app?

Reply:
No, only when the user "reset third party data" in the settings